index.html

```diff
@@ -1725,6 +1726,69 @@ <h3>Validity Period</h3>
 }
         </pre>
 
+        <h4>Representing Time</h4>
+
+        <p>
+When representing time values, implementers are urged to understand that times
+are not implemented in a uniform manner throughout the world. For example:
+        </p>
+
+        <ul>
+          <li>
+The observation of daylight savings time does not start or end on the same day
+and time, if at all, between various regions.
+          </li>
+          <li>
+The adoption of the Gregorian calendar did not happen in a single year; it was
+phased in over many years across various societies.
+          </li>
+          <li>
+Leap seconds might not be taken into account in all software systems, especially
```

Comment:
excellent to see this comment regarding leap seconds make its way into the spec.

Reply:
Leap seconds can affects highly sensitive systems that depend on the exact millisecond offset from the epoch. 

However, note that for most applications using ISO8601 serializations the only moment in time that is affected is the one second period of the leap second itself. That is, the moment *after* the most recent leap second can always be represented as `2017-01-01T00:00:00Z`, regardless of whether the system in question understands leap seconds.

Reply:
Fixed in e555089c2c41dd63b613b2aabbda8f73b12c9881.